Pod/Classes/PINRemoteImageManager.m

```diff
@@ -100,6 +99,8 @@ @interface PINRemoteImageManager () <PINURLSessionManagerDelegate>
     PINRemoteLock *_lock;
     NSOperationQueue *_concurrentOperationQueue;
     NSOperationQueue *_urlSessionTaskQueue;
+    PINAlternateRepresentationDelegate *_defaultAlternateRepresentationDelegate;
```

Comment:
Do we need this ivar? Seems like we could just have `_alternateRepDelegate` and assign it to a default in `init` without having a separate ivar just for the default delegate.